### PR TITLE
corrected cmake commands

### PR DESCRIPTION
**Pull request by blackliner**

### CODE REVIEW

**Comment by blackliner** [comment]
@googlebot I signed it!

**Comment by LebedevRI** [comment]
Please can you explain what correction does this perform?
Why is the old one wrong?

**Comment by blackliner** [comment]
Of course:
when i just tried out to cmake and make benchmark as it was described, i got errors. Which makes sense, because when you are in --/benchmark/build and do a cmake ../benchmark, there is no such folder --/benchmark/benchmark. I thought it was a spelling and acutally meant ../googletest, which makes sense, because you need to get the googletest targets first. After that you need to build benchmark via [cmake .. .]. You do not need to add the last dot to specify the current directory as output directory, but i think its more clear this way.
Cheers!

File: README.md [hunk 1]
```diff
@@ -69,7 +69,10 @@ $ git clone https://github.com/google/googletest.git benchmark/googletest
 $ mkdir build && cd build
 # Generate a Makefile with cmake.
 # Use cmake -G <generator> to generate a different file type.
-$ cmake ../benchmark
```
Comment (LebedevRI) [inline README.md]:
```suggestion
$ cmake ../
```

**Comment by LebedevRI** [comment]
> Of course:
> when i just tried out to cmake and make benchmark as it was described, i got errors. Which makes sense, because when you are in --/benchmark/build and do a cmake ../benchmark, there is no such folder --/benchmark/benchmark.

Yep, that's wrong.

> I thought it was a spelling and acutally meant ../googletest, which makes sense, because you need to get the googletest targets first.

Not, this is differently wrong.

>After that you need to build benchmark via [cmake .. .]. You do not need to add the last dot to specify the current directory as output directory, but i think its more clear this way.
> Cheers!

File: README.md (hunk 1 above)
Comment (blackliner) [reply README.md]: I assumed one needs to cmake gtest first, but this is already enough. cmake finds googletest sources and does its thing.

**Comment by blackliner** [comment]
Your last change should actually be enough. Changing it...

**Comment by blackliner** [comment]
done

**Comment by LebedevRI** [comment]
Now this looks ok.
I'm not sure about `CONTRIBUTORS`/`AUTHORS` files - when should they be updated.

**Comment by blackliner** [comment]
> I'm not sure about `CONTRIBUTORS`/`AUTHORS` files - when should they be updated.

I have no idea, and there is no need to rush anything for deleting a few characters 😄

**Comment by LebedevRI** [comment]
LG

**Pull request merged**